src/App.vue

```diff
@@ -666,10 +666,11 @@ export default class App extends Mixins(
     // NB: these were already validated in storeBootstrapItem()
     const header = filing.header
     const data = filing[header.name]
+    const status = header.status as FilingStatus
```

Comment:
I didn't notice this earlier, but why do you need assign a type here? Doesn't it already exist because the code knows the types of filing and header?

![image](https://github.com/user-attachments/assets/1b921705-a9d5-47c3-a919-79c171b7d9d0)

Reply:
Check that you have the following Extensions in VS Code:
- Vue-Official (v2.1.6)
- Vue 3 Support (by Eno Yao)

Reply:
Agree, will remove it